### ISSUE TITLE
Can I use this pipe with angular cli?

### ISSUE THREAD

**Issue by Invizible** [conclurer/markdown-to-html-pipe#3]
I use angular cli.
When I try to build project I get an error: 
```
/node_modules/markdown-to-html-pipe/src/markdown-to-html.pipe.ts:2:0 
Import assignment cannot be used when targeting ECMAScript 2015 modules. Consider using 'import * as ns from "mod"', 'import {a} from "mod"', 'import d from "mod"', or another module format instead.
```

**Comment by virgil-av** [comment]
If you  using CLI  first install:

npm i marked --save
npm i @types/marked

Make the import inside the pipe:

`import * as marked from 'marked';`

Even so the pipe will not really work as is, try this article: [Read this](https://www.reddit.com/r/Angular2/comments/4s2xda/keep_getting_errors_trying_to_import_marked_into/)

```
import {Pipe, PipeTransform} from '@angular/core';
import * as marked from 'marked';

@Pipe({
    name: 'markdown'
})
export class MarkdownPipe implements PipeTransform {
    transform(value: string) : string {
        if (value != null) {
            return marked(value);
        }
        return null;
    }
}
```

Using your pipe
Import your pipe into your component:
`import {MarkdownPipe} from "./MarkdownPipe.pipe";`

To use your pipe, you will need to bind it to the [innerHTML] property on an element in your template. When you use {{ blah }} inside an element, you're altering the text content, not the inner HTML.

```
<section id="markdown-in-here" [innerHTML]="defaultContent | markdown">
    <!-- Markdown will appear here in the DOM -->
</section>
```

**Comment by marvinscharle** [comment]
We will release an update for this pipe shortly. 

@virgil-av yes, you have to use `[innerHTML]` binding, since `{{ variable }}` will only support text nodes. This is documented in our README.

**Comment by vknabel** [comment]
@Invizible The fix is included in `markdown-to-html-pipe@1.2.0`

**Comment by Invizible** [comment]
Thanks
